var/spack/repos/builtin/packages/clp/package.py

```diff
@@ -15,7 +15,4 @@ class Clp(AutotoolsPackage):
 
     version('1.16.11', sha256='b525451423a9a09a043e6a13d9436e13e3ee7a7049f558ad41a110742fa65f39')
 
-    def configure_args(self):
-        return [
-            '--with-clp-datadir={0}/Data'.format(self.build_directory),
-        ]
+    build_directory = '.'
```

Comment:
```suggestion
    build_directory = 'spack-build'
```
This fixed it for me.

Reply:
Ping @molecuul